spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1580,3 +1594,35 @@ CDI 2.0 introduced the following Configurators interface:
 * <<observer_method_configurator>> for `ObserverMethod` configuration
 
 The container must provide implementation for all these configurators and make them available in matching container lifecycle events as defined in <<init_events>>.
+
+[[interception_proxy_factory]]
+
+=== Apply interceptor and decorators programmatically
+
+Contextual instances of bean having interceptors binding or defined decorators are provided by the container as a proxy.
```

Comment:
The term `proxy` is undefined from the spec point of view. The spec defines `client proxy` but it's not related to interceptors and decorators. I believe the wording should follow section `7.2. Container invocations and interception` and use terms like `contextual reference` and  `business method invocation`.